store/orm/orm.go

```diff
@@ -704,9 +705,45 @@ func (orm *ORM) DeleteStaleSessions(before time.Time) error {
 
 // BulkDeleteRuns removes runs given a query.
 func (orm *ORM) BulkDeleteRuns(bulkQuery *models.BulkDeleteRunRequest) error {
-	return orm.DB.
+	tx := orm.DB.Begin()
+	if tx.Error != nil {
+		return tx.Error
+	}
+
+	err := orm.DB.Exec(`
```

Comment:
I believe these should all be `tx.Exec(...)` instead of `orm.DB.Exec(...)` in order to use the transaction.